gswikichat/api.py

```diff
@@ -22,50 +21,49 @@ async def root():
 
 @app.get("/api")
 async def api(q):
+    print("query: ", q)
 
-    embedder, retriever, prompt_builder, llm, answer_builder
-
-    # query = "How many languages are there?"
     query = Document(content=q)
 
-    result = embedder.run([query])
+    queryEmbedded = embedder.run([query])
+    queryEmbedding = queryEmbedded['documents'][0].embedding
 
-    results = retriever.run(
-        query_embedding=list(result['documents'][0].embedding),
+    retrieverResults = retriever.run(
+        query_embedding=list(queryEmbedding),
         filters=None,
-        top_k=None,
+        top_k=3,
```

Comment:
We should change this line to `top_k=top_k`, where we pull in the base `top_k` from either a `config.json` or the UI